src/Sri.php

```diff
@@ -8,15 +8,22 @@ public function __construct()
     {
     }
 
-    public function html(string $path): string
+    public function html(string $path, bool $with_credentials = false): string
     {
         try {
             $integrity = $this->hash($path);
         } catch (\Exception $e) {
             return '';
         }
 
-        return "integrity={$integrity} crossorigin=anonymous";
+        $crossorigin = $with_credentials ? '=use-credentials' : '';
+
+        return "integrity={$integrity} crossorigin{$crossorigin}";
+    }
+
+    public function withCredentials(string $path): string
```

Comment:
I can't see the added value of this function. You added 11 characters to avoid writing 6.
The naming is also confusing, `withCredentials` but no credentials are passed down to the function.

```
html('/path/to/asset', true);
withCredentials('/path/to/asset');
```